ops/ansible/playbooks-ccs/launch_bfd-pipeline.yml

```diff
@@ -0,0 +1,50 @@
+---
+##
+# Configures and launches the ETL Server and data-pipeline app for CCS env
+##
+- name: Configure ETL Server
+  hosts: localhost
+  vars:
+    env: "test"
+
+  tasks:
+    
+    - name: Include env specific variables
+      include_vars:
+        dir: vars/{{ env }}
+      tags: [pre-ami, post-ami]
+      
+    - name: Apply Blue Button Data Pipeline Role
+      import_role:
+        name: bfd-pipeline
+      vars:
+        data_pipeline_dir: '/bluebutton-data-pipeline'
+        # Note: The `data_pipeline_jar` variable should have been provided by `--extra-vars` on the command line.
+        data_pipeline_jvm_args: "-Xmx{{ ((data_pipeline_ec2_instance_type_mem_mib | int) * 0.80) | int }}m -XX:+PreserveFramePointer"
+        data_pipeline_tmp_dir: "{{ data_pipeline_dir }}/tmp"
+        data_pipeline_loader_threads: "{{ data_pipeline_ec2_instance_type_vcpu * 25 }}"
+        data_pipeline_hicn_hash_iterations: "{{ vault_data_pipeline_hicn_hash_iterations }}"
+        data_pipeline_hicn_hash_pepper: "{{ vault_data_pipeline_hicn_hash_pepper }}"
+        data_pipeline_s3_bucket: "{{ vault_data_pipeline_s3_bucket }}"
+        # Primary (i.e. write) DB defined in `environments/<env>/group_vars/all/vault.yml`:
+        data_pipeline_db_url: "{{ vault_data_db_primary_url }}"
+        data_pipeline_db_username: "{{ vault_data_pipeline_db_username }}"
+        data_pipeline_db_password: "{{ vault_data_pipeline_db_username }}"
+        #data_pipeline_loader_threads: (see group_vars/all/main.yml 
+        
+    - name: Configure CloudWatch Logs Agent
```

Comment:
This shouldnt be needed here during launch. This task is already run during the pre-bake playbook.